### PR TITLE
MM-11447: Increases default sessions lengths to one year.

### PR DESCRIPTION
**Pull request by mkraft**
#### Summary
Increases default session lengths to one year.

#### Ticket Link
[MM-11447](https://mattermost.atlassian.net/browse/MM-11447)

#### Checklist
n/a

screen shot 2018-07-24 at 12 44 41 pm

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -34,9 +34,9 @@
         "EnableUserAccessTokens": false,
         "AllowCorsFrom": "",
         "AllowCookiesForSubdomains": false,
-        "SessionLengthWebInDays": 30,
-        "SessionLengthMobileInDays": 30,
-        "SessionLengthSSOInDays": 30,
+        "SessionLengthWebInDays": 365,
```
Comment (lieut-data) [inline config/default.json]: I think we have to update `model/config.go` too?

https://github.com/mattermost/mattermost-server/blob/master/model/config.go#L376-L386

Comment (mkraft) [reply config/default.json]: @lieut-data You're right thanks.

**Comment by Ulisses-champ83** [comment]
See"" because l can control the Console.  That's why. I don't want to
change my password developed. Not even with smart pasword work.. but I find
the way.

u.e.m

**Comment by Ulisses-champ83** [comment]
Response soon. What I do. I have in on my hand the azk my the if l use
2016"2017 and the new confi..
On bottom page. Two option.
Exports ..or default

u.e.m

**Comment by crspeller** [comment]
Taking this to the security team. Let's hold off merging for now.

**Pull request closed without merging**